server/middlewares/validateUser.js

```diff
@@ -0,0 +1,38 @@
+import { userSchema } from '../schemas/userSchema';
+/**
+ * This class contains all methods
+ * required to validate
+ * sent data from the UI.
```

Comment:
The data is sent from the API consumer instead. It might be Postman, Insomnia, or SwaggerUI. Change the comment.

Reply:
Done